### PR TITLE
Log and report operating project details when return permission denied error

### PR DESCRIPTION
**Pull request by evankanderson**
# Summary

By reporting the project ID the user is operating on, they may realize they have selected the wrong project and be able to fix the problem themselves.

## Change Type

***Mark the type of change your PR introduces:***

- [x] Bug fix (resolves an issue without affecting existing features)
- [ ] Feature (adds new functionality without breaking changes)
- [ ] Breaking change (may impact existing functionalities or require documentation updates)
- [ ] Documentation (updates or additions to documentation)
- [ ] Refactoring or test improvements (no bug fixes or new functionality)

# Testing

N/A

# Review Checklist:

- [x] Reviewed my own code for quality and clarity.
- [x] Added comments to complex or tricky code sections.
- [x] Updated any affected documentation.
- [ ] Included tests that validate the fix or feature.
- [x] Checked that related changes are merged.

### CODE REVIEW

**Review by puerco — DISMISSED** [review]

**Comment by puerco** [comment]
ugh linter is sad :/

**Comment by evankanderson** [comment]
And the tests caught me up, too.  Fixed!

File: internal/controlplane/handlers_authz_test.go [hunk 1]
```diff
@@ -232,7 +233,9 @@ func TestProjectAuthorizationInterceptor(t *testing.T) {
 					ID: projectID,
 				},
 			},
-			rpcErr: util.UserVisibleError(codes.PermissionDenied, "user is not authorized to perform this operation"),
+			rpcErr: util.UserVisibleError(
+				codes.PermissionDenied,
+				fmt.Sprintf("user is not authorized to perform this operation on project %q", projectID)),
```
Comment (puerco) [inline internal/controlplane/handlers_authz_test.go:238]: At some point we may want to catalogue these?

**Review by puerco — APPROVED** [review]

File: internal/controlplane/handlers_authz_test.go (hunk 1 above)
Comment (JAORMX) [reply internal/controlplane/handlers_authz_test.go:238]: this would be handy!

**Review by teodor-yanev — APPROVED** [review]

**Pull request merged**